xbmc/cores/IPlayer.h

```diff
@@ -159,9 +159,6 @@ enum EINTERLACEMETHOD
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL=14,
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL_HALF=15,
   VS_INTERLACEMETHOD_DEINTERLACE_HALF=16,
-  VS_INTERLACEMETHOD_DXVA_BOB = 17,
```

Comment:
MAX seems to be used by the XML loader as a safeguard against reading incorrect values. But max itself isn't used as a valid option, so I think this is redundant...